### PR TITLE
Make clear that automation run button skips conditions + remove word "execute" from UI

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

1. Make it clear that the trigger/run button for an automation only runs the actions, but skips the conditions.
2. Remove the word "execute" from automation and scripts as advised by the Microsoft and Google style guides, since it might carry negative connotations.

Renamed:
Automation editor: "Trigger" => "Run Actions"
Script editor: "Execute" => "Run Script"
Script Entity Row: "Execute" => "Run" (same value as used by Service Entity Row)

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion: https://github.com/home-assistant/frontend/issues/8201
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by frenck** [comment]
That is actually intentional. It is for running purely the action part of an automation.

**Comment by spacegaier** [comment]
> 
> 
> That is actually intentional. It is for running purely the action part of an automation.

Then we need to make that clear to the user, although I personally do not really get the reasoning behind that option. Perhaps we should offer both, otherwise you cannot test your automation conditions.

**Comment by frenck** [comment]
Well, I would say a TRIGGER button could be helpful in some cases (e.g., when you want to debug your conditions?). But the EXECUTE button executes the script logic of an automation (the action part).

Looking at my own setup, 95% of my automations won't be testable/running able with a TRIGGER, as they will never pass conditions manually.

**Comment by spacegaier** [comment]
What do you think about this:
1. Renaming current button to "Execute Actions".
2. Adding a second "Trigger" button?

That would make it clear and would prevent others from having to debug through HA core to realize that `skip_condition` is suddenly true 😆.

**Comment by bramkragten** [comment]
I think an additional trigger button could make sense in some places :-)

**Comment by spacegaier** [comment]
@bramkragten Forgot to update the ticket: I chatted with Frank about it a few days ago and his leaning was:
1. Rename the button to "Trigger Actions" (avoid the word "Execute" in the frontend completely as advised in many style guides).
2. Not add a second button as a) you can test an automation via the dev service tools and tell the service to use the conditions and b) the new upcoming automation trace/debug feature will cover that in some way.

What do you think?

**Comment by bramkragten** [comment]
`Run action` then?

**Comment by frenck** [comment]
Yes "Run" is the advised word of choice by both the style guides of Microsoft & Google. Adding "action(s)" makes it more clear on what it runs.

**Comment by spacegaier** [comment]
@bramkragten @frenck I updated the PR description with the new changes I made based on our discussion. Let me know if that meets your intentions.

File: src/panels/config/script/ha-script-editor.ts [hunk 1]
```diff
@@ -299,12 +299,12 @@ export class HaScriptEditor extends KeyboardShortcutMixin(LitElement) {
                                     <mwc-button
                                       @click=${this._runScript}
                                       title="${this.hass.localize(
-                                        "ui.panel.config.script.picker.activate_script"
```
Comment (spacegaier) [inline src/panels/config/script/ha-script-editor.ts:302]: This key `activate_script` did not exist in the translations to begin with.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**